src/Faker/Provider/ro_RO/Text.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Faker\Provider\ro_RO;
+
+class Text extends \Faker\Provider\Text
+{
+    /**
+     * From uk.wikisource.org
+     * Text available under Creative Commons Attribution-ShareAlike 3.0 Unported
```

Comment:
Since the work is in the public domain (the author died in 1918), this can be removed.